src/components/data_submission/DataSubmissionNihAnvilUse.js

```diff
@@ -0,0 +1,87 @@
+import {div, h, h2} from 'react-hyperscript-helpers';
+import {useState} from 'react';
+import {FormField, FormFieldTypes, FormValidators} from '../forms/forms';
+
+
+export default function DataSubmissionNihAnvilUse(props) {
+  const { onChange } = props;
+  const [nihAnvilUse, setNihAnvilUse] = useState();
+
+  const I_DID = 'I did';
+  const I_WILL = 'I will';
+  const NO = 'No';
+
+  return h(div, {
+    style: {
+      padding: '50px 0',
+      maxWidth: 800,
+      margin: 'auto'
+    }
+  }, [
+    h2('NIH and AnVIL use'),
+    h(FormField, {
+      id: 'nihAnvilUse',
+      title: 'Will you or did you submit data to the NIH?',
+      type: FormFieldTypes.RADIO,
+      selectOptions: [
+        {label: I_DID, value: I_DID},
+        {label: I_WILL, value: I_WILL},
+        {label: NO, value: NO},
+      ],
+      dependentFormFields: [
+        {id: 'submittingToAnvil', type: FormFieldTypes.RADIO, shouldClear: ({value}) => value === I_DID},
+        {id: 'dbGaPPhsID', type: FormFieldTypes.TEXT},
+        {id: 'dbGaPStudyRegistrationName', type: FormFieldTypes.TEXT},
+        {id: 'embargoReleaseDate', type: FormFieldTypes.TEXT},
+        {id: 'sequencingCenter', type: FormFieldTypes.TEXT},
+      ],
+      validators: [FormValidators.REQUIRED],
+      onChange,
+      setFormValueParent: setNihAnvilUse
```

Comment:
My ticket requires different questions to be rendered based on the option selected for the `nihAnvilUse`, but the `formValue` state variable is contained within the `FormField` itself. To address this, I'm also tracking the value of `nihAnvilUse` in the `DataSubmissionNihAnvilUse` component.

Past Attempts:
 I tried experimenting getting the value from `formData.nihAnvilUse`, but the changes didn't register in my component since the same formData object is being used. I also tried having formData be a state variable, and in onChange, I would deepClone the current contents of formData, set the given key value pair, and call setFormData with the new object. However, this caused the hook in `DataSubmissionStudyInformation` to be called way too often. 

Current Approach:
Currently, I am passing `setNihAnvilUse` as an optional `setFormValueParent` prop to FormField, which is called with the new form field value in `onFormInputChange`. This ensures that when the `formValue` is changed in the FormField, it is also changed in this component